dist/index.js

```diff
@@ -35417,23 +35420,42 @@ class GitHub {
 			issue_number: prNumber,
 		};
 
-		const commentMessage =
-		"Here is a list of everyone that appears to have contributed to this PR and any linked issues:\n\n" +
+		let commentMessage = "The following accounts have interacted with this PR and/or linked issues. I will continue to update these lists as activity occurs, or manually ask me to refresh this list by adding the `props-bot` label.\n\n";
+
+		if ( contributorsList['unlinked'].length > 0 ) {
+			commentMessage += "## Unlinked Accounts\n\n" +
+				"The following contributors have not linked their GitHub and WordPress.org accounts: @" + contributorsList['unlinked'].join(', @') + ".\n\n" +
+				"Please [read how to link your accounts](https://make.wordpress.org/core/2020/03/19/associating-github-accounts-with-wordpress-org-profiles/) here to ensure your work is properly credited in WordPress releases.\n\n";
```

Comment:
"here" should probably be within the linked text or removed

Reply:
Removed is better in my eyes.
```suggestion
				"Please [read how to link your accounts](https://make.wordpress.org/core/2020/03/19/associating-github-accounts-with-wordpress-org-profiles/) to ensure your work is properly credited in WordPress releases.\n\n";
```